src/webrtc/callFeed.ts

```diff
@@ -55,18 +65,26 @@ type EventHandlerMap = {
     [CallFeedEvent.LocalVolumeChanged]: (localVolume: number) => void;
     [CallFeedEvent.VolumeChanged]: (volume: number) => void;
     [CallFeedEvent.Speaking]: (speaking: boolean) => void;
+    [CallFeedEvent.VoiceActivityThresholdChanged]: (threshold: number) => void;
+    [CallFeedEvent.VADMuteStateChanged]: (VADMuted: boolean) => void;
 };
-
 export class CallFeed extends TypedEventEmitter<CallFeedEvent, EventHandlerMap> {
     public stream: MediaStream;
+    public secondStream: MediaStream;
+    public audioDelay: DelayNode;
     public sdpMetadataStreamId: string;
     public userId: string;
     public purpose: SDPStreamMetadataPurpose;
     public speakingVolumeSamples: number[];
+    public voiceActivityThreshold: number;
+    public setVoiceActivityDetectionMute: (muted: boolean) => void;
```

Comment:
I think you started working on replacing this with `VADMuteStateChanged` but did not finish it?